stack.yaml

```diff
@@ -88,9 +88,9 @@ Resources:
               Override: true
               Value: 'Vapor API Docs'
         SecurityHeadersConfig:
-          # ContentSecurityPolicy:
-          #   ContentSecurityPolicy: default-src 'none'; script-src 'self'; img-src 'self'; style-src 'self'; font-src 'self'
-          #   Override: false
+          ContentSecurityPolicy:
+            ContentSecurityPolicy: default-src 'none'; script-src 'self' 'unsafe-inline'; img-src 'self'; style-src 'self' 'unsafe-inline'; font-src 'self'; connect-src 'self'; base-uri 'self'; form-action 'self'
```

Comment:
We absolutely can't use unsafe inline here for scripts. I'm hesitant to even use it for styles and they should really be pulled out into a separate file

Reply:
Thanks for the clarification!